### PR TITLE
SCUMM: Add Hebrew font for zak and MM

### PR DESCRIPTION
**Pull request by ShaharAriel**
The replacement character tables override English letter A-Z (+1).
The replacement character tables have been created using the make-charset-fontdata tool.

### CODE REVIEW

File: engines/scumm/charset-fontdata.cpp [hunk 1]
```diff
@@ -549,6 +549,67 @@ static const byte russCharsetDataV2[] = {
 	125, 233,
 	126, 231,
 };
+// Hebrew Zak and MM font map.
```
Comment (bluegr) [inline engines/scumm/charset-fontdata.cpp:552]:
```suggestion
    // Hebrew Zak and MM font map.
```

File: engines/scumm/charset-fontdata.cpp [hunk 2]
```diff
@@ -549,6 +549,67 @@ static const byte russCharsetDataV2[] = {
...
+            85, 20,
+            86, 21,
+            87, 22,
+            88, 23,
+            89, 24,
+            90, 25,
+            91, 26,
+    };
+
+//Hebrew Zak and MM font data
```
Comment (bluegr) [inline engines/scumm/charset-fontdata.cpp:583]:
```suggestion
    // Hebrew Zak and MM font data
```

File: engines/scumm/charset-fontdata.cpp [hunk 3]
```diff
@@ -628,6 +689,13 @@ CharsetRendererV2::CharsetRendererV2(ScummEngine *vm, Common::Language language)
 			_fontPtr = russianCharsetDataV2;
 		}
 		break;
+	case Common::HE_ISR:
+		if (((_vm->_game.id == GID_MANIAC) || (_vm->_game.id == GID_ZAK)) && (_vm->_game.version == 2)) {
```
Comment (bluegr) [inline engines/scumm/charset-fontdata.cpp:693]:
```suggestion
		if ((_vm->_game.id == GID_MANIAC || _vm->_game.id == GID_ZAK) && _vm->_game.version == 2) {
```
Makes the code easier to read

**Comment by ShaharAriel** [comment]
Why this https://github.com/scummvm/scummvm/pull/3856/commits/257ef41e732d296a7970a319beb04a045b752cdc commit failed, the error is:
```
 Error: Failed to download from mirror set:
 https://github.com/fribidi/fribidi/archive/v1.0.11.tar.gz: failed: status code 502
```

**Comment by ShaharAriel** [comment]
Why this https://github.com/scummvm/scummvm/pull/3856/commits/257ef41e732d296a7970a319beb04a045b752cdc commit failed, the error is:
```
 Error: Failed to download from mirror set:
 https://github.com/fribidi/fribidi/archive/v1.0.11.tar.gz: failed: status code 502
```

**Comment by bluegr** [comment]
Thanks for your work on this! Merging

**Pull request merged**